docs/docsite/rst/community/committer_guidelines.rst

```diff
@@ -4,153 +4,73 @@
 Committers Guidelines
 *********************
 
-These are the guidelines for people with commit privileges on the Ansible GitHub repository. Committers are essentially acting as members of the Ansible Core team, although not necessarily as employees of Ansible and Red Hat. Please read the guidelines before you commit.
+These are the guidelines for people with commit privileges on the repositories in the ansible and ansible-collections GitHub organizations.
 
-These guidelines apply to everyone. At the same time, this ISN'T a process document. So just use good judgment. You've been given commit access because we trust your judgment.
```

Comment:
Could we change this so for collections we talk about the community *first*, as they are the main set of people.

Reply:
@gundalow PTAL. I returned it in place with `community`  instead of `we` or could you say in other words what you mean?

Reply:
Something along the lines of

Committers of `Ansible collections <https://github.com/ansible-collections/>`_ are members of the community or Ansible Engineering. 

Not core

Reply:
@gundalow done, PTAL